sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
@mssfang @conniey 
This change was/is not correct. You should NOT have been using current for core dependencies outside of the core pipeline, that's the whole purpose of unreleased dependencies. I'll be fixing this in https://github.com/Azure/azure-sdk-for-java/pull/8866 so please do not make further updates or start a new PR for this.